### ISSUE TITLE
conflict in training neural network model

### ISSUE THREAD

**Issue by Bingsnow** [CCQC/PES-Learn#28]
i have trained the same the data two times with neural network model ,each time RMSE varies how is it possible ?

**Comment by sgoodlett** [comment]
Are you passing "rseed" as an option to the input object? Neural networks are initialized with a random weight matrix, and so that can lead to differing results. Additionally, if the sampling routine you are using is random based, that would also give different results.

* `rseed`  
  **Description:**  Global random seed. Used for initializing hyperparameter optimization iterations, random training set sampling.
    * **Type**: int
    * **Default**: None
    * **Possible values**: Any integer

**Comment by Bingsnow** [comment]
no ,i will do that and test
